docs/uzl_omics.md

```diff
@@ -0,0 +1,32 @@
+# nf-core/configs: UzL OMICS Cluster Configuration
+
+The rnaseq nf-core pipeline has been successfully configured for use on the UzL OMICS cluster at the University of Luebeck.
+Implementation of nf-core pipelines for use on the UzL OMICS is in process. 
+
+To use, run the pipeline with `-profile uzl_omics`. This will download and launch the `uzl_omics.config` which has been pre-configured with a setup suitable for the UzL OMICS cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Nextflow and Singularity using the environment module system on UzL OMICS cluster. You can do this by issuing the commands below:
+
+```bash
+module load nextflow
+module load singularity
+```
+
+nf-core requires a Nextflow version 22.10.1 or higher, so you have to intall a more recent version first.
+
+For Nextflow versions newer than 23.07.0-edge, it is necessary to explicitly mount the home directory using the command:
```

Comment:
There seems to be some buggy modules not being happy without a writable `$HOME`, is this because of this or some other thing?

Reply:
It seems I made the last comments there ;)

Was the outcome that the singularity setup was intentionally broken (not allowing mount controls)? I guess that would limit to `$HOME` if that was the case, but then again, I'd definitely try raising it for a chance to have it fixed first.

Reply:
Could you please explain what you mean and how we can get it fixed?

The singularity is installed by the HPC maintainers, and we are currently unsure why this problem occurs when we do not explicitly ask to mount `$HOME` using the global variable `NXF_SINGULARITY_HOME_MOUNT`.

In our understanding, it is rather a Nextflow related issue, because previous versions (e.g. 22.10.1) of Nextflow worked fine without this variable decoration.  Starting from version 23.07.0-edge, nextflow documentation says that the default is set to false when it comes to mounting the home directory.

Reply:
As was mentioned in the slack thread, it is very likely a good idea to reach out to your systems people and let them now that their singularity configuration (disallowing user binds) makes things more difficulty for you.

I obviously don't know how your systems are set up exactly, and maybe everything is under users' home directories - then making sure nextflow doesn't disable home binds would be good enough (although the user experience is lousy as it currently requires setting that environment variable). But obviously that doesn't cover anything run outside of the home directories.

A setup that doesn't have singularity stopping if a bind is provided could use `singularity.autoMounts` to pass the binds nextflow knows of that would be required (although in some cases with links going back and forth, it might be a good idea to help with specifying some binds explicitly).

So, to sum up, I'd say the singularity configuration is weird if it doesn't allow the user specificying bind mounts (which it seemed from the slack thread), and I'd generally suggest trying to fix that since it breaks a lot of use cases.

Reply:
We will work on fixing this issue. For now I have edited the documentation file and erased this part, to not cause any conflicts.